maps/tether/tether_shuttles.dm

```diff
@@ -63,6 +63,7 @@
 //
 // The backup tether shuttle uses experimental engines and can degrade and/or crash!
 //
+/* //Disabling the crash mechanics per request
```

Comment:
I would like to revisit this mechanic someday, but today is not that day.

If we ever re-enable the Virgo wilderness, the shuttle crash mechanic should be possible to trigger under certain conditions. The Virgo wilderness was a cool local exploration thing.